src/Bicep.Core.Samples/Files/InvalidResources_CRLF/main.diagnostics.bicep

```diff
@@ -131,20 +131,17 @@ resource baz 'Microsoft.Foo/foos@2020-02-02-alpha' = {
   name: 'test'
   id: 2
 //@[2:4) [BCP073 (Error)] The property "id" is read-only. Expressions cannot be assigned to read-only properties. |id|
-//@[6:7) [BCP036 (Error)] The property "id" expected a value of type "string" but the provided value is of type "int". |2|
   type: 'hello'
 //@[2:6) [BCP073 (Error)] The property "type" is read-only. Expressions cannot be assigned to read-only properties. |type|
-//@[8:15) [BCP036 (Error)] The property "type" expected a value of type "'Microsoft.Foo/foos'" but the provided value is of type "'hello'". |'hello'|
   apiVersion: true
 //@[2:12) [BCP073 (Error)] The property "apiVersion" is read-only. Expressions cannot be assigned to read-only properties. |apiVersion|
-//@[14:18) [BCP036 (Error)] The property "apiVersion" expected a value of type "'2020-02-02-alpha'" but the provided value is of type "bool". |true|
 }
 
 resource badDepends 'Microsoft.Foo/foos@2020-02-02-alpha' = {
   name: 'test'
   dependsOn: [
     baz.id
-//@[4:10) [BCP034 (Error)] The enclosing array expected an item of type "resource", but the provided item was of type "string". |baz.id|
+//@[8:10) [BCP053 (Error)] The type "Microsoft.Foo/foos@2020-02-02-alpha" does not contain property "id". Available properties include "eTag", "extendedLocation", "identity", "kind", "location", "managedBy", "managedByExtended", "name", "plan", "properties", "scale", "sku", "tags", "zones". |id|

```

Comment:
This seems like a regression. Any idea why this is happening?

Reply:
+1